pkg/cli/flags.go

```diff
@@ -363,6 +363,7 @@ func init() {
 		StringFlag(f, &startCtx.listeningURLFile, cliflags.ListeningURLFile, startCtx.listeningURLFile)
 
 		StringFlag(f, &startCtx.pidFile, cliflags.PIDFile, startCtx.pidFile)
+		StringFlag(f, &startCtx.geosLocation, cliflags.GEOSLocation, "")
```

Comment:
```go
StringFlag(f, &startCtx.geosLocation, cliflags.GEOSLocation, startCtx.geosLocation)
```

then initialize `startCtx.geosLocation` in `context.go` in the same directory. Thanks